modules/blocks/utils.js

```diff
@@ -270,8 +270,25 @@ Utils.prototype.loadBlockByHeight = function(height, cb, tx) {
 	(tx || library.db).blocks
 		.loadBlocksOffset(height, height + 1)
 		.then(rows => {
-			const blocks = self.readDbRows(rows);
-			return setImmediate(cb, null, blocks[0]);
+			// Normalize block
+			const block = modules.blocks.utils.readDbRows(rows)[0];
```

Comment:
why not `self`?

Reply:
I just copied the same code from other method, where it was used.